language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/Update.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.javaapi.data.codegen;
+
+import com.daml.ledger.javaapi.data.Command;
+import java.util.function.Function;
+
+public abstract class Update<U> {
```

Comment:
```suggestion
public abstract class Update<U> extends HasCommands {
```

As discussed offline.